src/sidebar/components/test/group-list-item-out-of-scope-test.js

```diff
@@ -28,27 +26,23 @@ describe('GroupListItemOutOfScope', () => {
       .first()
       .simulate('click');
 
-  before(() => {
+  beforeEach(() => {
+    fakeAnalytics = {
+      track: sinon.stub(),
+      events,
+    };
     fakeGroupListItemCommon = {
       orgName: sinon.stub(),
       trackViewGroupActivity: sinon.stub(),
     };
 
-    GroupListItemOutOfScope = proxyquire('../group-list-item-out-of-scope', {
-      // Use same instance of Preact module in tests and mocked module.
-      // See https://robertknight.me.uk/posts/browserify-dependency-mocking/
-      preact,
-
+    GroupListItemOutOfScope.$imports.$mock({
       '../util/group-list-item-common': fakeGroupListItemCommon,
-      '@noCallThru': true,
     });
   });
 
-  beforeEach(() => {
-    fakeAnalytics = {
-      track: sinon.stub(),
-      events,
-    };
+  afterEach(() => {
+    GroupListItemOutOfScope.$imports.$restore();
```

Comment:
This is for cleaning up the mock objects between tests.

Reply:
The docs note that Jest would do this automatically for us. This test runner looks pretty nice. Is that something you think we might move to in the future?

Reply:
Maybe. As I understand it, Jest currently runs only under Node and [not in a real browser](https://github.com/facebook/jest/issues/139). That is OK for many web apps where a fake browser environment like [JSDOM](https://github.com/jsdom/jsdom) is good enough for running tests. For the client I'm not sure if it will be adequate.